isb_lib/geome_adapter.py

```diff
@@ -360,18 +360,14 @@ def reloadThing(thing):
     identifier = igsn_lib.normalize(thing.id)
     return loadThing(identifier, thing.tcreated)
 
-def _set_source_on_core_record(core_record: typing.Dict):
-    core_record["source"] = "GEOME"
 
 def reparseAsCoreRecord(thing: Thing) -> typing.List[typing.Dict]:
     _validateResolvedContent(thing)
     core_records = []
-    transformer = isamples_metadata.GEOMETransformer.GEOMETransformer(thing.resolved_content)
+    transformer = isamples_metadata.GEOMETransformer.GEOMETransformer(thing.resolved_content, thing.tcreated)
```

Comment:
This seeds the transformer with the `last_modified` of `thing.tcreated`, which makes its way to the solr document last updated time, which is what we use to do partial index updates.